pkg/client/client.go

```diff
@@ -297,3 +297,16 @@ func DefaultOpenShiftUserAgent() string {
 	version = seg[0]
 	return fmt.Sprintf("%s/%s (%s/%s) openshift/%s", path.Base(os.Args[0]), version, runtime.GOOS, runtime.GOARCH, commit)
 }
+
+type HybridClient struct {
```

Comment:
Ok, I think that our [Client](https://github.com/openshift/origin/blob/43d027cdcccfbbd10b48ea498ca8e0b9c51cb63c/pkg/client/client.go#L238-L241) should include the `ExtensionsClient` and then internally mix the clients like we already do for `scale` and `delete`.